controller-server/src/main/java/com/yahoo/vespa/hosted/controller/restapi/application/ApplicationApiHandler.java

```diff
@@ -990,6 +1006,21 @@ private Slime toSlime(Tenant tenant, HttpRequest request, boolean listApplicatio
         return slime;
     }
 
+    private Slime toSlime(URI propertyUri, URI contactsUri, URI issueCreationUri, List<? extends List<? extends User>> contacts) {
+        Slime slime = new Slime();
+        Cursor root = slime.setObject();
+        root.setString("propertyUri", propertyUri.toString());
+        root.setString("contactsUri", contactsUri.toString());
+        root.setString("issueCreationUri", issueCreationUri.toString());
+        Cursor lists = root.setArray("contacts");
+        for (List<? extends User> contactList : contacts) {
+            Cursor list = lists.addArray();
+            for (User contact : contactList)
+                list.addString(contact.toString());
```

Comment:
Should use `username()` to avoid unintended consequences of changing `toString`.

Reply:
This is only for display in a table somewhere, and I wanted exactly the toString, since it includes the type of contact, in the subclass of User which we end up using in our implementation. 

I guess I could add a `displayName()` instead, then?